plugins/inputs/logfile/tail/tail.go

```diff
@@ -390,7 +391,7 @@ func (tail *Tail) tailFileSync() {
 						if errReadLine == nil {
 							tail.sendLine(line, tail.curOffset)
 						} else {
-							break
+							return
```

Comment:
🤔  because of the return [a couple lines down](https://github.com/aws/amazon-cloudwatch-agent/pull/452/files#diff-b4cbda30ad21153305e4658dca8a5844580e52b12544b5147237a3315d185284R400), this doesn't change any behavior, right?

Reply:
There is only 1 for loop in the `tailFileSync()` method.
And there is no code after the end of the for loop.
So a `break` is effectively the same as `return` here.
It caught my eye and confused me why `break` was used on this line, when `return` is used _everywhere else_ within `tailFileSync()`.

So this change is just for consistency and to reduce my confusion.